config/routes.rb

```diff
@@ -1,5 +1,14 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-  root "homes#index"
+  root "monsters#index"
+  resources :monsters
```

Comment:
Where possible we should be explicit with the available routes - this is also set up for a regular controller which it looks like all of your views are handled in React, so we don't need a regular controller